contrib/vagrant/provision-util.sh

```diff
@@ -128,6 +128,9 @@ os::provision::set-os-env() {
   fi
   os::provision::set-bash-env "${origin_root}" "${config_root}" \
 "/root/${file_target}"
+
+  # Make symlinks to the bash completions for the openshift commands
+  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
```

Comment:
Seeing this error for openshift tab completion: '-bash: _get_comp_words_by_ref: command not found' (on F21 vagrant instance)